doajtest/fixtures/__init__.py

```diff
@@ -2,7 +2,8 @@
 from applications import ApplicationFixtureFactory
 from editors import EditorGroupFixtureFactory
 from accounts import AccountFixtureFactory
-from article import ArticleFixtureFactory
+from article_doajxml import DoajXmlArticleFixtureFactory
```

Comment:
I noticed that you changed `ArticleFixtureFactory` for `DoajXmlArticleFixtureFactory`.  I think that makes sense for when you are asking the factory for DOAJXML documents, but many calls, like `make_incoming_api_source` are unrelated to DOAJXML, so I think we should keep the `ArticleFixtureFactory` for those, and only use the `DoajXmlArticleFixtureFactory` for actual XML fixtures.

Reply:
I removed `ArticleFixtureFactory` at all and added `DoajXmlArticleFixtureFactory` and `CrossrefArticleFixtureFactory` instead. All tests related to articles are run for both doaj and crossref. It might be useful if we would like to make further changes to other parts of the systems to be sure nothing is broken here. I've found only 3 tests that are totally unrelated to the schema (they do not need schema as a parameter at all) and these are separate.